winmerge.sls

```diff
@@ -1,12 +1,15 @@
+{% set version = '2.16.0' %}
```

Comment:
Rather than remove old versions, we should make this a list of available versions.
```
(% set versions = ['2.16.0', '2.14.0']  %)
```